### DISCUSSION TITLE
how i can use RegEx - lookarounds in nuclei templates

### DISCUSSION THREAD

**Discussion by ayadim** [projectdiscovery/nuclei#3396] [Q&A] [answered]
hi,
am trying to create a regular expression that extracts variable names without including keywords.
the solution is using a regular expression with lookarounds and here is my regex 
```
(?i)((?<=var)|(?<=let)|(?<=const))\\s+([a-z_$][0-9a-z_$]*)(?<!\\s*=)
```
but something caused an error during the template validation. 

here a practice example : https://regex101.com/r/Q56MfP/1 

regards.

**Comment by ehsandeep** [comment, marked as answer]
https://github.com/projectdiscovery/nuclei/issues/244#issuecomment-678801691

**Reply by ayadim** [reply]
so lookahead is not allowed in Go !! 
thank you
